<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame#frame{
	background-color: qlineargradient(spread:pad, x1:1.072045, y1:0.124, x2:0.368, y2:1, stop:0.227273 rgba(80, 214, 255, 155), stop:0.806818 rgba(112, 32,213,191));
	border-radius:10px;
}
QPushButton#Closed{
	background-color: rgba(0, 0,0, 0);
	color:rgba(10,15,155,255);
	
	border-radius:5px;
}
QPushButton#Closed:hover{
	color:rgba(250,223,11,255);
}
QPushButton#Closed:pressed{
	padding-left:1px;
	padding-top:1px;
	background-color:rgba(150,123,111,255);
}
QLineEdit#PlaylistLink{
	background-color:rgba(0,0,0,0);
	border:2px solid rgba(0,0,0,0);
	border-bottom-color: rgba(1,100,100,255);
	color:rgb(0,0,0);
	padding-bottom:7px;
}


QPushButton#Select_Home{
	background-color: rgba(0, 0,0, 0);
	color:rgba(10,15,155,255);
	border-radius:5px;
}

QPushButton#Select_Home:hover{
	color:rgba(250,223,11,255);
}

QPushButton#Select_Home:pressed{
	padding-left:1px;
	padding-top:1px;
	background-color:rgba(150,123,111,255);
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>260</width>
      <height>350</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>13</y>
       <width>260</width>
       <height>41</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777204</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Mistral</family>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Spotify Downloader</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="Closed">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="PlaylistLink">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>200</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="whatsThis">
      <string notr="true">Enter Spotify playlist Link</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="placeholderText">
      <string>Enter Spotify Playlist Link</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="PlaylistMsg">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>NSimSun</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Playlist Code : </string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>6</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Created By Surenjanath</string>
     </property>
    </widget>
    <widget class="QLabel" name="SongName">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>163</y>
       <width>201</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>NSimSun</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Status :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="statusMsg">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="PlaylistMsg_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>NSimSun</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Song Name  : </string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="CounterLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>NSimSun</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Counter : Songs downloaded 0</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Select_Home">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>300</y>
       <width>201</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Imprint MT Shadow</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Follow me on Medium</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
